site/packages/aa-signers/arcana-auth/authenticate.md

```diff
@@ -0,0 +1,65 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: ArcanaAuthSigner • authenticate
+  - - meta
+    - name: description
+      content: Overview of the authenticate method on ArcanaAuthSigner
+  - - meta
+    - property: og:description
+      content: Overview of the authenticate method on ArcanaAuthSigner
+---
+
+# authenticate
+
+`authenticate` is a method on the `ArcanaAuthSigner` that leverages the `Arcana` Auth Web SDK to authenticate a user.
+
+This method must be called before accessing the other methods available on the `ArcanaAuthSigner`, such as signing messages or typed data or accessing user details.
+
+## Usage
+
+::: code-group
+
+```ts [example.ts]
+// [!code focus:99]
+import { ArcanaAuthSigner } from "@alchemy/aa-signers";
+// Register app through Arcana Developer Dashboard to get clientId
+// ARCANA_AUTH_CLIENTID = "xar_live_nnnnnnnnnn"
+const newArcanaAuthSigner = new ArcanaAuthSigner({
+  clientId: ARCANA_AUTH_CLIENTID,
+});
+// or
+// import { AuthProvider } from "@arcana/auth";
+// const inner = new AuthProvider ("xar_live_nnnn");
+// const newArcanaAuthSigner = new ArcanaAuthSigner({inner});
+const getUserInfo = await newArcanaAuthSigner.authenticate();
+```
+
+:::
+
+## Returns
+
+### `Promise<UserInfo>`
+
+A Promise containing the `UserInfo`, an object with the following fields:
+
+- `address: string | null` -- the EoA account address associated with the authenticated user's wallet.
+- `email: string | null` -- email address of the authenticated user.
+- `id: string | null` -- the decentralized user identifier.
+- `loginToken: string` -- JWT token returned after the user authenticates.
+- `loginType: Logins | passwordless` -- login provider type or passwordless used for authentication (ex. `[{ Logins: "google" | "github" | "discord" | "twitch" | "twitter" | "aws" | "firebase" | "steam" }]`).
+- `name: string` -- user name associated with the email id
+- `picture: string` -- url pointing to the user profile image
+- `publicKey: string` -- public key associated with the user account
+
+See [Arcana Auth SDK Reference Guide](https://authsdk-ref-guide.netlify.app/interfaces/userinfo) for details.
+
+## Parameters
+
+### `clientId`: Unique app identifier assigned after app registration via the Arcana Developer Dashboard
+
+or
+
+### `inner`: An AuthProvifer object. For field details, see [AuthProvider constructor](https://authsdk-ref-guide.netlify.app/classes/authprovider#constructor).
```

Comment:
wait @mmjee can you review this

Reply:
The AuthProvider object can have multiple optional parameters and those are listed in Arcana SDK Ref guide under the constructor section.  Therefore, we linked it. What is the issue here @avasisht23 ?
cc: @mmjee

Reply:
oh my original comment was made about doc not listing every parameter, but might have changed since my comment. githubu ui says my comment isn't outdated so just making sure the parameters are corr/ect on this method.

Reply:
if it looks good to you, then great! i'll merge it in.

Reply:
I think it should be OK.